<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serveWindow</class>
 <widget class="QMainWindow" name="serveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务员</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="svSelButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查</string>
    </property>
   </widget>
   <widget class="QPushButton" name="svDelButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删</string>
    </property>
   </widget>
   <widget class="QPushButton" name="svAddButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>增</string>
    </property>
   </widget>
   <widget class="QPushButton" name="svPwdButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="svChButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>改</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>471</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="svTableTab">
     <attribute name="title">
      <string>桌面管理</string>
     </attribute>
     <widget class="QTableView" name="svTableTabView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="svDishesTab">
     <attribute name="title">
      <string>点菜管理</string>
     </attribute>
     <widget class="QTableView" name="svDishesTabView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="svMoneyTab">
     <attribute name="title">
      <string>账单管理</string>
     </attribute>
     <widget class="QTableView" name="svMoneyTabView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>441</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="svTabNumSumLab">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>340</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>总计 :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="svTabNumLab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>输入要查询的桌号 :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="svTabNumEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="svPayButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>结账</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="svNameLab">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户名 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="svNameEdit">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="svPpwd1Lab">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>原密码 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="svPwd1Edit">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="svPwd2Lab">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>新密码 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="svPwd2Edit">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="svPwd3Lab">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>新密码 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="svPwd3Edit">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="svSubButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="serveMenu">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="svTableAction"/>
    <addaction name="svDishesAction"/>
    <addaction name="svPayAction"/>
   </widget>
   <widget class="QMenu" name="pwdMenu">
    <property name="title">
     <string>密码管理</string>
    </property>
    <addaction name="svPwdAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="svCancelAction"/>
   </widget>
   <addaction name="serveMenu"/>
   <addaction name="pwdMenu"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="svTableAction">
   <property name="text">
    <string>桌位管理</string>
   </property>
  </action>
  <action name="svDishesAction">
   <property name="text">
    <string>点菜管理</string>
   </property>
  </action>
  <action name="svPayAction">
   <property name="text">
    <string>结账管理</string>
   </property>
  </action>
  <action name="svPwdAction">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="svCancelAction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
